### ISSUE TITLE
Output of curl-config --ca now surrounded by quotes; breaks other builds

### ISSUE THREAD

**Issue by ryandesign** [curl/curl#15041]
### I did this

```
curl-config --ca
```

### I expected the following

```
/opt/local/share/curl/curl-ca-bundle.crt
```

but I got this instead:

```
"/opt/local/share/curl/curl-ca-bundle.crt"
```

### curl/libcurl version

```
curl 8.10.1 (x86_64-apple-darwin21.6.0) libcurl/8.10.1 OpenSSL/3.3.2 zlib/1.3.1 brotli/1.1.0 zstd/1.5.6 libidn2/2.3.7 libpsl/0.21.5 nghttp2/1.63.0
Release-Date: 2024-09-18
Protocols: dict file ftp ftps gopher gophers http https imap imaps ipfs ipns mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli HSTS HTTP2 HTTPS-proxy IDN IPv6 Largefile libz NTLM PSL SSL threadsafe TLS-SRP UnixSockets zstd
```

### operating system

```
Darwin redacted 21.6.0 Darwin Kernel Version 21.6.0: Wed Apr 24 06:02:02 PDT 2024; root:xnu-8020.240.18.708.4~1/RELEASE_X86_64 x86_64
```

**Comment by ryandesign** [comment]
It looks like this got broken in curl 8.8.0 by fa69b41c7790fab86fd363242c81d8ef2e89e183.

This causes other software to fail to build, [for example osslsigncode](https://trac.macports.org/ticket/70919).

**Comment by vszakats** [comment]
I believe this was fixed in 9f56bb608ecfbb8978c6cb72a04d9e8b23162d82 via #14681, sub-commit 01119bb95f908826ea755cd5f7df719bc02ddab1.

Can you retest with current master?

**Comment by ryandesign** [comment]
Thanks, you're right, it's solved in master.
